uber/config.py

```diff
@@ -189,7 +205,7 @@ def _unrepr(d):
 
 c.BADGE_PRICES = _config['badge_prices']
 for _opt, _val in chain(_config.items(), c.BADGE_PRICES.items()):
-    if not isinstance(_val, dict):
+    if not isinstance(_val, dict) and not hasattr(c, _opt.upper()):
         setattr(c, _opt.upper(), _val)
```

Comment:
also, I tested this carefully on the staging server first (good thing, since I discovered the crash) so this works on our existing deploys just fine.